smartsim/_core/control/controller.py

```diff
@@ -110,6 +162,11 @@ def start(
         self._jobs.kill_on_interrupt = kill_on_interrupt
         # register custom signal handler for ^C (SIGINT)
         signal.signal(signal.SIGINT, self._jobs.signal_interrupt)
+
+        # have job manager launch a telemetry monitor if any jobs are run
+        if not self._jobs._on_start_hook:  # pylint: disable=protected-access
+            self._jobs.add_job_onstart_callback(create_telmon_callback(exp_path))
```

Comment:
Still not in love with this solution, but willing to call it good enough for now!! @MattToast and/or @ankona remember to write up a ticket to address in future

Reply:
I think i'm abusing the callbacks here, anyway. Why don't I just call the start method instead of add a callback?

Reply:
I've removed the callbacks altogether.